### ISSUE TITLE
一些细节问题求助

### ISSUE THREAD

**Issue by 1a1a1a1a1a1a1a1** [yufei1900/DAST_segmentation#4]
loss_adv = 0
        D_out = model_D[0](feat_target)
        loss_adv += bce_loss(D_out, torch.FloatTensor(D_out.data.size()).fill_(source_label).to(device))
        D_out = model_D[1](F.softmax(pred_target, dim=1))
        loss_adv += bce_loss(D_out, torch.FloatTensor(D_out.data.size()).fill_(source_label).to(device))
        loss_adv = loss_adv*0.01
        loss_adv.backward()
        
        optimizer.step()
 
请问一下您这里为什么使用的target的特征但是是源域的label,这里我有一点没有理解。
